### ISSUE TITLE
[Code Refactor]Improve code climate of ImageToPdfFragment

### ISSUE THREAD

**Issue by dutta14** [Swati4star/Images-to-PDF#898]
https://codeclimate.com/github/Swati4star/Images-to-PDF/code?sort=maintainability

ImageToPdfFragment has a rating of F. Bring this up to B or higher.

**Comment by rubenxcjesus** [comment]
Can I work on this issue?

**Comment by Swati4star** [comment]
@dutta14  Are you working on this issue?

**Comment by dutta14** [comment]
@Swati4star , yep I have started working on this, it is a little more tightly coupled compared to the text to PDF feature so it is taking some more time.

**Comment by dutta14** [comment]
@Swati4star and @rubenxcjesus , I have not been able to contribute a lot of time in the past two weeks and am booked for the next 2 weeks, so if you want, you can pick this one up.

**Comment by dutta14** [comment]
@Swati4star , I can start working on this again in a week if no one has picked this up yet.

**Comment by Swati4star** [comment]
@dutta14 This issue is up for grabs.

**Comment by dutta14** [comment]
@Swati4star  Working on this starting today.

**Comment by McModknower** [comment]
Can i work an this issue?

**Comment by Swati4star** [comment]
Sure @McModknower
